core/cap-0035.md

```diff
@@ -182,15 +237,41 @@ in many cases should be reserved for licensed entities (like a transfer agent)
 holding the issuer credentials and aware of responsibilities under the law of 
 the jurisdiction of the affected party and asset. 
  
+### Reusing the AUTH_REVOCABLE flag
+The account `AUTH_REVOCABLE` flag allows the issuer to indicate that it has
+control over the use of the asset on the network. By including the
+`AUTH_REVOCABLE` flag in account flags, account owners may review the
+revocability of an asset issued by the issuer and have the choice to avoid this
+type of asset if they object to the implied trust in the issuer. Clawback is
+another form of an issuer revoking use of an asset with fine control over the

```

Comment:
I agree with the similarities described here, but I think it's worth making it clearer that `ClawbackOp` effectively reduces the total number of an asset that exists in the network, whereas `AllowTrustOp` doesn't. So in some ways 
 ClawbackOp` might seem more definitive in theory, although not necessarily in practice.